rfcs/20210129-optimize-pd-creation/README.md

```diff
@@ -0,0 +1,262 @@
+# Optimization of Primitive Descriptor Creation
+
+## Motivation
+
+There are three majors steps to create a oneDNN primitive:
+1. Create operation descriptor
+2. Create primitive descriptor
+3. Create primitive
+
+In frameworks, all the three steps are performed every time when a user executes
+an operation. The last two steps are expensive hence they can significantly
+affect performance of the operation.
+
+oneDNN recently introduced a cache for primitives to significantly reduce primitive
```

Comment:
There are many many notions in this RFC which can make reading it a bit confusing. I would suggest to emphasize that the entity the RFC is based on is explicitly called `primitive cache`, and use it in as many places as possible.

Reply:
Thanks for suggesting a better writing for the places you find confusing, it definitely makes the RFC better.